Makefile

```diff
@@ -153,13 +153,27 @@ docker-test-unit: $(DOCKER_CACHE)
 	@$(DOCKER_ENV) make test-unit
 	@chmod -R 0755 $<
 
-.PHONY: docker-test-integration
-docker-test-integration: .coverage
+.PHONY: build-test-image
+build-test-image:
 	@echo "===> Building Antrea Integration Test Docker image <==="
 ifneq ($(NO_PULL),)
 	docker build -t antrea/test -f build/images/test/Dockerfile $(DOCKER_BUILD_ARGS) .
 else
 	docker build --pull -t antrea/test -f build/images/test/Dockerfile $(DOCKER_BUILD_ARGS) .
+endif
+
+HAS_TEST_IMAGE = $(shell docker images antrea/test | grep antrea/test > /dev/null && echo "yes")
+.PHONY: docker-test-integration
+docker-test-integration: .coverage
+ifneq ($(CHECK_IMAGE),true)
+	@echo "===> You can set 'CHECK_IMAGE=true' to check and use existing image if it does exist <==="
+	make build-test-image
```

Comment:
in which situation it should skip building the test image and just rerun the test? The test image includes the code to test, if we don't update image, how to test change?
And docker cache can skip 1~10 steps if the image has been built once, and only copy the updated code to image, which should just take very few seconds. I wonder why we need to set CHECK_IMAGE.

Reply:
Hi @tnqn , The test codes are mounted during `docker run`. Only `build/images/scripts/*` and  `build/images/test/test-integration` are included in the image, they are quite stable per my understanding. This is just a convenient option for developer to choose if they know image build can be skipped directly. I added this because it's unable to build this image locally even I set the GOPROXY.  Default behavior is not changed, it should have no side affect.